save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestSuitesService.kt

```diff
@@ -190,36 +185,19 @@ class TestSuitesService(
         ) {
             "Expected that we remove all test suites related to a single execution at once"
         }
-        executionIds.forEach { executionId ->
-            log.debug { "Delete link between execution $executionId and test suites" }
-            lnkExecutionTestSuiteService.deleteByExecution(executionId)
-        }
+        executionService.markAsObsoleteByIds(executionIds)
```

Comment:
What do you mean by "obsolete" ? Legacy? Deleted? Why we need to have obsolete executions? How we get obsolete executions? I do not get this logic...

Reply:
Ah, just moved logic to the Service?

Reply:
> I do not get this logic...

It's existed logic of execution archiving 

> Ah, just moved existing logic to the Service?

yes

Reply:
will move this logic to `ExecutionStatus#updateExecutionStatus` to hide this method inside in service

Reply:
removed